frontend-ssr/redux/exercise-submissions/submissions.reducer.ts

```diff
@@ -0,0 +1,49 @@
+import { SubmissionState } from './types';
+
+const initialState: SubmissionState = {
+  submissions: [],
+};
+export const submissionReducer = (
+  state = initialState,
+  action: { type: string; payload: any; id: string},
+): SubmissionState => {
+  switch (action.type) {
+    case 'GET_SUBMISSIONS':
+      return {
+        ...state.submissions,
+        submissions: action.payload,
+      };
+    case 'ADD_SUBMISSION':
+      return {
+        submissions: [...state.submissions, action.payload],
```

Comment:
Let's add a parameter here which is the index where to insert the submission. It will come in handy

Reply:
Also, let's merge ADD & Get. There's no point in having both of them. We can just keep the `/ADD` one and make it in such a way that we can insert multiple values at once.